src/routes/safe/components/Settings/ManageOwners/AddOwnerModal/screens/Review/index.jsx

```diff
@@ -42,15 +43,15 @@ const ReviewAddOwner = ({
     let isCurrent = true
     const estimateGas = async () => {
       const web3 = getWeb3()
-      const { fromWei, toBN } = web3.utils
+      const { fromWei } = web3.utils
       const safeInstance = await getGnosisSafeInstanceAt(safeAddress)
 
       const txData = safeInstance.contract.methods
         .addOwnerWithThreshold(values.ownerAddress, values.threshold)
         .encodeABI()
       const estimatedGasCosts = await estimateTxGasCosts(safeAddress, safeAddress, txData)
 
-      const gasCostsAsEth = fromWei(toBN(estimatedGasCosts), 'ether')
+      const gasCostsAsEth = fromWei(BigNumber(estimatedGasCosts).toString(), 'ether')
```

Comment:
why did you add it here? In the ticket it's mentioned is that problem happens only transaction rows with transactions with value ~1bn ether